aneris/utils.py

```diff
@@ -212,7 +213,7 @@ def subtract_regions_from_world(df, name=None, base_year='2015', threshold=5e-2)
         threshold below which to set values to 0
     """
     # make global only global (not global + sum of regions)
-    check_null(df, name)
+    # check_null(df, name)
```

Comment:
why does this check need to be removed?

Reply:
see comment in commit https://github.com/iiasa/aneris/pull/18/commits/6c1d85c5b75ce47ad63964a7a80d341925daa3d9

>@gidden commenting `check_null(df, name)` solve the KeyError here (see previous commit with global_ar6 test). However, I'm not fully sure what exactly the nature of the problem was and why this line was implemented in the first place. Is commeting this out reasonable?

Reply:
See new comments for updated reasoning here